### PR TITLE
.plan-hydro-ng-dev: For installers

### PR DESCRIPTION
**Pull request by estreby**
To manage installers for HYDRO

### CODE REVIEW

File: scripts.d/.plan-hydro-ng-dev [hunk 1]
```diff
@@ -22,7 +22,12 @@ s-proj:#6.3.2
...
 
+[ = deb,9
+    s-salome-ng/hydro:s9-dev
+]
+
 [ = deb,10
 	s-cmake:skip
-        s-hydro/ng:master
+    s-hydro/ng:master
+    s-salome-ng/hydro:s10-dev
```
Comment (TurboGit) [inline scripts.d/.plan-hydro-ng-dev:32]: hydro-s10-dev?

File: scripts.d/.plan-hydro-ng-dev [hunk 2]
```diff
@@ -22,7 +22,12 @@ s-proj:#6.3.2
 
 #s-cloudcompare:#2.10.3edf
 
+[ = deb,9
+    s-salome-ng/hydro:s9-dev
```
Comment (TurboGit) [inline scripts.d/.plan-hydro-ng-dev:26]: :hydro-s9-dev ?

**Review by TurboGit — COMMENTED** [review]
Wondering, if you want hydro to be part of the installer name I think my comments are valid.

**Comment by estreby** [comment]
Update with hydro in installer name

**Pull request merged**